pythermalcomfort/models/bfu_occupational.py

```diff
@@ -0,0 +1,323 @@
+from __future__ import annotations
+
+import numpy as np
+
+from pythermalcomfort.classes_input import BFU_occupational_Inputs
+from pythermalcomfort.classes_return import BFU_occupational as BFUOccupationalResult
+
+
+def bfu_occupational(
+    tdb: float | list[float] | np.ndarray,
+    rh: float | list[float] | np.ndarray,
+    tr: float | list[float] | np.ndarray | None = None,
+    clo: float | list[float] | np.ndarray = 0.28,
+    met: float | list[float] | np.ndarray = 125.0,
+    fan_velocity: float | list[float] | np.ndarray = 3.5,
+    fan_off_velocity: float | list[float] | np.ndarray = 0.2,
+    position: str = "standing",
+    activity: str = "walk",
+    speed: float | list[float] | np.ndarray = 1.66,
+    fixed_sweat_rate_lph: float | list[float] | np.ndarray | None = 1.0,
+    body_surface_area: float | list[float] | np.ndarray = 1.81,
+    round_output: bool = True,
+) -> BFUOccupationalResult:
+    """Compute the potential benefit of fans in an occupational context using the
+    validated model.
+
+    Parameters
+    ----------
+    tdb : float or list of floats
+        Dry-bulb air temperature, :math:`°C`.
+    rh : float or list of floats
+        Relative humidity, :math:`%`.
+    tr : float or list of floats, optional
+        Mean radiant temperature, :math:`°C`. If omitted or equal to ``999`` it is assumed equal to ``tdb``.
+    clo : float or list of floats, optional
+        Clothing insulation, [clo]. Defaults to 0.28.
+    met : float or list of floats, optional
+        Metabolic rate, [W/m²]. Defaults to 125.0.
+    fan_velocity : float or list of floats, optional
```

Comment:
should not be the fan speed at the body? The fan speed at the fan is not very relevant since then we need to enter the distance from the body to the fan and calculate the resulting speed at the body